mmpose/datasets/transforms/pose3d_transforms.py

```diff
@@ -44,11 +44,11 @@ class RandomFlipAroundRoot(BaseTransform):
     """
 
     def __init__(self,
-                 keypoints_flip_cfg,
-                 target_flip_cfg,
-                 flip_prob=0.5,
-                 flip_camera=False,
-                 flip_label=False):
+                 keypoints_flip_cfg: dict,
+                 target_flip_cfg: dict,
```

Comment:
Do you think it would be better to pass arguments directly instead of using a dict?

Reply:
> Do you think it would be better to pass arguments directly instead of using a dict?

I think it's better to create a new PR if we decide to list all parameters at the constructor, it will affect all MotionBERT and VideoPose configs.

Reply:
> I think it's better to create a new PR if we decide to list all parameters at the constructor, it will affect all MotionBERT and VideoPose configs.

All right. Just keep it this time